lib/judges/baza.rb

```diff
@@ -38,14 +38,15 @@
 # Copyright:: Copyright (c) 2024 Yegor Bugayenko
 # License:: MIT
 class Judges::Baza
-  def initialize(host, port, token, ssl: true, timeout: 30, retries: 3, loog: Loog::NULL)
+  def initialize(host, port, token, ssl: true, timeout: 30, retries: 3, loog: Loog::NULL, enable_compression: true)
```

Comment:
@tank-bohr try to avoid compound names: https://www.yegor256.com/2015/01/12/compound-name-is-code-smell.html